app/controllers/pages_controller.rb

```diff
@@ -25,6 +29,8 @@ def cookies
   end
 
   def show
+    render_not_found && return if directly_requesting_index_page?
+
```

Comment:
As we only have `index.md` pages in these story directories, would we get the same effect by moving `returners/index.md` up a level and renaming to `returners.md`?

Reply:
I'm blaming 3pm Friday for not seeing this 😬 In order to rename those files we would either need the changes to redirect `/index` renders or extra routes to account for it though, so I'll merge this down as-is, then update the content repo and raise another PR to pull out these changes.